tests/unit/utils/pipeline/graph/d3-graph-util-test.js

```diff
@@ -55,7 +55,7 @@ module('Unit | Utility | pipeline-graph | d3-graph-util', function () {
         { name: 'job1' },
         { name: 'job2', displayName: '0123456789' },
         { name: 'job3' },
-        { name: '~sd@1:foo', displayName: '0123456789/0123456789' }
+        { name: '~sd@1:foo', remotename: '0123456789/0123456789' }
```

Comment:
```suggestion
        { name: '~sd@1:foo', remoteName: '0123456789/0123456789' }
```
The typo has no adverse effects on the test